rn/Teacher/src/canvas-api/httpClient.js

```diff
@@ -27,7 +27,8 @@ type Method = 'GET' | 'POST' | 'PUT' | 'DELETE'
 type Body = null | void | string | Object | FormData | Blob | ArrayBuffer
 
 export function resolveUrl (url: string, config: ApiConfig) {
-  const baseURL = (config.baseURL || getSession().baseURL || '').replace(/\/?$/, '')
+  // FIXME: hardcoded url. MBL-13344
+  const baseURL = (config.baseURL || getSession().baseURL || 'https://canvas.instructure.com').replace(/\/?$/, '')
```

Comment:
hmm should we hold off on merging this until that ticket is done? Probably won't make a difference.. probably going to be an error either way if it's a bug in real life, right?

Reply:
It sounded like it would be hard to track down the root cause, and I'd really like to have a nightly build pass soon.